src/OrchardCore/OrchardCore.Email.Abstractions/SmtpResult.cs

```diff
@@ -19,7 +19,7 @@ public class SmtpResult
         public IEnumerable<LocalizedString> Errors { get; protected set; }
 
         /// <summary>
-        /// Get or sets thet response text from the SMTP server.
+        /// Get or sets the response text from the SMTP server.
```

Comment:
Thanks for fixing your own typo ;)

I would also suggest to remove the `if` term in the below comment `Whether if the operation succeeded or not.`

But already merged, no problem I know that you prefer smaller PRs that are easier to manage ;)

Reply:
> I would also suggest to remove the if term in the below comment Whether if the operation succeeded or not.

Seems this added by Seb :)